microreactors/hpmr/steady/tests

```diff
@@ -0,0 +1,44 @@
+[Tests]
+  # Syntax checks
+  [coupled_multiphysics_syntax]
+    type = RunApp
+    input = 'neutronics_eigenvalue.i'
+    check_input = True
+    executable_pattern = 'blue_crab*|dire_wolf*|fenix*'
+  []
+  [neutronics_only_syntax]
+    type = RunApp
+    input = 'neutronics_eigenvalue.i'
+    check_input = True
+    executable_pattern = 'griffin*|blue_crab*|dire_wolf*|fenix*'
+    cli_args = "MultiApps/active='' Transfers/active=''"
+  []
+  [bison_mp_syntax]
+    type = RunApp
+    input = 'thermal_ss.i'
+    check_input = True
+    executable_pattern = 'blue_crab*|dire_wolf*|bison*|fenix*'
+  []
+  [bison_only_syntax]
+    type = RunApp
+    input = 'thermal_ss.i'
+    check_input = True
+    executable_pattern = 'blue_crab*|dire_wolf*|bison*|fenix*'
+    cli_args = "MultiApps/active='' Transfers/active=''"
+  []
+  [sockeye_only_syntax]
+    type = RunApp
+    input = 'heatpipe_vapor_only.i'
+    check_input = True
+    executable_pattern = 'dire_wolf*|sockeye*|fenix*'
+  []
+
+  # Regression tests with CSV output over a minimum amount of time steps/computation
+  [sockeye_only]
+    prereq = sockeye_only_syntax
+    type = CSVDiff
+    csvdiff = 'heatpipe_vapor_only_out.csv'
+    input = 'heatpipe_vapor_only.i'
+    executable_pattern = 'dire_wolf*|sockeye*|fenix*'
+  []
```

Comment:
add the test for griffin with min_parallel = 192
it wont run... for now

Reply:
added